source/js/opening-hours.bindings.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @file
+ * Menu component binding.
+ */
+(function (Drupal) {
+  'use strict';
+
+  Drupal.behaviors.gentBaseLoadOpeningHours = {
+    attach: function (context, settings) {
+      this.initMonthWidgets();
+      this.determineAccordionVisibility();
+    },
+
+    /**
+     * Determine whether or not the accordion should be expanded on page load
```

Comment:
Can't you solve this in the templates by putting aria-expanded on true?

Reply:
I could, but I'm not sure if it's a good idea to put `aria-expanded` on `.opening-hours-accordion` since there will still be visible content if the value is `false`.. ?

Reply:
I don't quite follow, which value could be false?

Setting aria-expanded=true on the first one should keep the accordion expanded.
As a non-js fallback, all accordions are expanded by default, the init function then collapses them based on the expanded attribute. (at least, that's how I remember it)

Reply:
@delrueba, after some more thorough investigation I recall why I implemented this the way I did.. 

The accordion is initialized before the data from the API is retrieved and rendered in the DOM. That means the accordion height is not correctly calculated if we just add the `aria-expanded` attribute. Instead, we need to wait for the data to be rendered, then open the accordion manually...

(I can give you a quick demo of this on my local environment, should the comment above make no sense at all)

Reply:
Yeah, no, makes sense.